source/_integrations/tailscale.markdown

```diff
@@ -28,8 +28,8 @@ the state of the devices in your Tailscale VPN network (Tailnet).
 This integration **DOES NOT** make your Home Assistant accessible via
 Tailscale VPN remotely!
 
-If you want to access your Home Assistant instance remotely, you will
-need to install Tailscale itself on your own. For instructions on how to do
+If you want to access your Home Assistant instance remotely check out the [TailScale Addon](https://github.com/hassio-addons/addon-tailscale) available under the 
```

Comment:
The Home Assistant project does not support or endorse the use of third-party add-ons, integrations or cards.

For that reason, I'm going to friendly decline this addition.

../Frenck